packages/theme-data/src/colorSchemes/darkGray/unresolvedRoles.js

```diff
@@ -3,13 +3,15 @@ import mapKeys from "../../utils/mapKeys";
 import darkBlueTheme from "../darkBlue";
 import system from "./system";
 import tooltip from "./components/tooltip";
+import slider from "./components/slider";
```

Comment:
a bit of a nitpick, but the components look to be alphabetized in the other `unresolvedRoles` files.